src/main/scala/com/avast/scala/hashes/Sha256.scala

```diff
@@ -14,5 +16,6 @@ case class Sha256(bytes: Array[Byte]) {
 }
 
 object Sha256 {
-  def apply(hex: String): Sha256 = Sha256(hex2bytes(hex))
+  private val bytesLength = 32
+  def apply(hexOrBase64: String): Sha256 = Sha256(if (hexOrBase64.length == 2 * bytesLength) hex2bytes(hexOrBase64) else base642bytes(hexOrBase64))
```

Comment:
Is it theoretically possible that the BASE64 encoding will match the length of hex-encoding?

Reply:
I had the same question and it looks like the size of random 32 bytes is always a string with 44 characters. I will include the test to this PR.

Reply:
> Is it theoretically possible that the BASE64 encoding will match the length of hex-encoding

IMO it could, for very short bytestrings, because base64 adds those `=` paddings, but that shouldn't be our case, since we encode sha256, which is 32 bytes